src/librustc_trans/trans/type_of.rs

```diff
@@ -223,7 +224,14 @@ pub fn sizing_type_of<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>, t: Ty<'tcx>) -> Typ
 
         ty::TyStruct(..) => {
             if t.is_simd(cx.tcx()) {
-                let llet = type_of(cx, t.simd_type(cx.tcx()));
+                let e = t.simd_type(cx.tcx());
+                println!("sizing_type_of: simd: {}", e);
```

Comment:
This seems to be a stray debugging println!, which is hit while compiling libcore.